internal/core/event/event.go

```diff
@@ -0,0 +1,44 @@
+package event
+
+import (
+	"encoding/json"
+
+	"github.com/polygonid/sh-id-platform/pkg/pubsub"
+)
+
+const (
+	CreateCredentialEvent = "createCredentialEvent" // CreateCredentialEvent create credential event
+	CreateConnectionEvent = "createConnectionEvent" // CreateConnectionEvent create connection MyEvent
```

Comment:
should be event?

Reply:
It could be ID or Topic? Probably the best could be just CreateCredential (without the event word)  but then it collides with the struct name.